src/ipc/dbus/ipc-dbus.h

```diff
@@ -978,6 +977,73 @@ class DBusIPCAttachedPropertyFactory : public IPCAttachedPropertyFactoryBase
 };
 
 }
+
+
+class InterfacePropertyHandlerBase {
+public:
+
+    QString generateObjectPath(const QString& parentPath) {
```

Comment:
Nitpick: "our" coding style says that the opening brace of a method should go on a new line (here and below).